gameboard.js

```diff
@@ -0,0 +1,66 @@
+var Gameboard = function (size) {
+  if (size % 2 !== 0) {
+    size ++;
+  }
+  this.size = size;
+  this.piecesArray = new PiecesArray(Math.pow(size, 2)).shuffledPieces();
+  this.currentPieces = [];
+  this.matchedPieces = 0;
+};
+
+Gameboard.prototype.buildBoard = function(){
+  var index = 0;
+
+  for (numRow = 0; numRow < this.size; numRow++) {
+    $(".gameboard").append("<tr>");
+
+    for (numCell = 0; numCell < this.size; numCell++) {
+      var piece = this.piecesArray[index];
+      $(".gameboard").append("<td><div class='piece' data-piece-type='" + piece + "'>" + piece + "</div></td>");
+      index++;
+    }
+
+    $(".gameboard").append("</tr>");
```

Comment:
...rather than try to wrap them. To use the latter approach, you should just concatenate strings and then only do a single `append()`.